### DISCUSSION TITLE
Could not get my account lmysticl

### DISCUSSION THREAD

**Discussion by putrenkov** [community/community#140119] [Code Security] [answered]
### Select Topic Area

Question

### Body

On my email do not receive email to get my account!

**Comment by ThisIs-Developer** [comment, marked as answer]
hey @putrenkov,

if you're not receiving the email to access your account, try these steps:

1. **check your spam/junk folder**.
2. **ensure your email is correct** in the account settings.
3. **request a new email link** from the login page.
4. **wait a bit**, as sometimes emails can be delayed.

if you still don’t receive anything, consider reaching out to GitHub support directly for further assistance.

---
if this helps, please mark it as the answer! 😪

**Reply by putrenkov** [reply]
:)It all was done the email did not come to my email thats all email is right spam box empty I wait more then 7 hours. Why my account just block password. I got 2 factr Auth! And if even  the pass was week!  Just block my work

**Reply by putrenkov** [reply]
Thanks for fixing now I tried for field email and it work properly! <3

**Reply by ThisIs-Developer** [reply]
That's great to hear, @putrenkov! 🎉  

Please mark it as the **answer** and close the discussion if everything is working properly now.

**Reply by ghostinhershell** [reply]
Hi @putrenkov 👋🏾,It looks like @ThisIs-Developer's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

**Reply by putrenkov** [reply]
@ghostinhershell sure np!

**Comment by ibrahimkhled** [comment]
🙄
